nx-dev/ui-common/src/lib/plugin-card.tsx

```diff
@@ -35,18 +49,97 @@ export function PluginCard({
           className="focus:outline-none"
         >
           <span className="absolute inset-0" aria-hidden="true" />
-          <p className="line-clamp-3 mb-6 text-sm">{description}</p>
+          <p className="line-clamp-3 mb-8 text-sm">{description}</p>
 
           {isOfficial ? (
             <span
               title="Official plugins are maintained by the Nx Team"
-              className="bg-green-500 absolute bottom-3 right-4 rounded-full px-3 py-0.5 text-xs font-medium capitalize text-white"
+              className="border bg-green-50 border-green-300 text-green-600 dark:border-green-900 dark:bg-green-900/30 dark:text-green-400 absolute bottom-3 right-4 rounded-full px-3 py-0.5 text-xs font-medium capitalize"
             >
               Official
             </span>
-          ) : null}
+          ) : (
+            <div className="absolute bottom-3 right-0 left-0 flex justify-between px-3 py-0.5 text-xs font-medium capitalize">
+              <div className="mx-1">
+                {lastPublishedWidget(lastPublishedDate)}
+              </div>
+              <div className="mx-1">{npmDownloadsWidget(npmDownloads)}</div>
+              <div className="mx-1">{githubStarsWidget(githubStars)}</div>
+              <div className="mx-1">{nxVersionWidget(nxVersion)}</div>
+            </div>
+          )}
         </a>
       </div>
     </div>
   );
 }
+
+function lastPublishedWidget(lastPublishedDate: string | undefined) {
+  if (!lastPublishedDate) {
+    return <div className="w-16"></div>;
+  }
+  return (
+    <abbr title="Most Recent Release Date">
```

Comment:
Can we show these titles as `<small>` when touch devices are detected?
Right now on mobile is not clear what these are.